iron-core/src/main/java/io/axway/iron/core/StoreManagerBuilder.java

```diff
@@ -22,7 +22,7 @@ static StoreManagerBuilder newStoreManagerBuilder(String name, Properties proper
 
     StoreManagerBuilder withEntityClass(Class<?> entityClass);
 
-    <T> StoreManagerBuilder withCommandClass(Class<? extends Command<T>> commandClass);
+    <T> StoreManagerBuilder withCommandClass(Class<? extends Command<?>> commandClass);
```

Comment:
Now that you're using ?, the <T> at the start can be removed

Reply:
Nice